lib/core/flip_card_core.dart

```diff
@@ -5,6 +5,8 @@ import 'package:flutter/cupertino.dart';
 
 import '../screen/asset_image_name.dart';
 
+enum FlipCardCoreEvent { resetCard, equalCard }
```

Comment:
스트림 방향에 따라 암묵적으로 정해지는 코딩 스탠다드가 있는데... 
여기서는 XXX -> UI 로 전달되는 이벤트는 보통 State로 네이밍합니다.

```suggestion
enum FlipCardCoreState { resetCard, equalCard }
```

Reply:
@devsoupe UI Event는 State로 네이밍 기억하겠습니다.